packages/providers/providers.yaml

```diff
@@ -114,13 +110,13 @@ adobe-umapi:
             title: ''
             description: ''
             automated: true
-
 adyen:
     display_name: Adyen
     categories:
         - payment
     auth_mode: OAUTH2
-    authorization_url: https://ca-${connectionConfig.environment}.adyen.com/ca/ca/oauth/connect.shtml
+    authorization_url: >-
```

Comment:
not a huge fan of this formatting to be honest, especially when there is no new lines